func_adl_xAOD/cms/miniaod/query_ast_visitor.py

```diff
@@ -0,0 +1,48 @@
+from func_adl_xAOD.common.ast_to_cpp_translator import query_ast_visitor
+from func_adl_xAOD.common.statement import book_ttree, set_var, ttree_fill
+from func_adl_xAOD.common.util_scope import gc_scope_top_level
+import func_adl_xAOD.common.cpp_types as ctyp
+import func_adl_xAOD.common.cpp_representation as crep
+from func_adl_xAOD.common.event_collections import EventCollectionSpecification
+import ast
+import func_adl_xAOD.common.statement as statement
+class book_cms_miniaod_ttree(book_ttree):
```

Comment:
Add two blank lines after your imports.

In general - have you run these through the "black" formatter? That will fix up these sorts of formatting errors on the fly without you having to think.

Reply:
I have not... I will do it right now.

Reply:
I ran the 'black' formatter on the repository. Most things it did is to change single quotes into double quotes on most of the files. It even broke some of the tests... I think I will reformat the code by hand for now. Sorry for the trouble!

Reply:
Ok - sounds great! We should add "format by black" as a future item to fix things up. What you are proposing sounds great!